dependencies/pom.xml

```diff
@@ -110,6 +110,7 @@
         <version.lib.narayana>5.9.3.Final</version.lib.narayana>
         <version.lib.netty>4.1.58.Final</version.lib.netty>
         <version.lib.oci-java-sdk-objectstorage>1.28.0</version.lib.oci-java-sdk-objectstorage>
```

Comment:
Upgrade to latest version (as in master)

Reply:
fixed